low-level/files/opt/seagate/sspl/prerequisites/sspl_dev_deploy.sh

```diff
@@ -0,0 +1,484 @@
+            TARGET_BUILD="$2"
+            shift 2 ;;
+        -L|--local_rpms_path)
+            [ -z "$2" ] && echo "ERROR: Local RPMS not provided" && exit 1;
+            RPMS_PATH="$2"
+            shift 2 ;;
+        --i|--bmc_ip)
+            [ -z "$2" ] && echo "ERROR: BMC IP not provided" && exit 1;
+            BMC_IP="$2"
+            shift 2 ;;
+        --u|--bmc_user)
+            [ -z "$2" ] && echo "ERROR: BMC user not provided" && exit 1;
+            BMC_USER="$2"
+            shift 2 ;;
+        --p|--bmc_passwd)
+            [ -z "$2" ] && echo "ERROR: BMC password not provided" && exit 1;
+            BMC_PASSWD="$2"
+            shift 2 ;;
+        --St|--storage_type)
+            [ -z "$2" ] && echo "ERROR: Storage type not provided" && exit 1;
+            STORAGE_TYPE="$2"
+            shift 2 ;;
+        --Sr|--server_type)
+            [ -z "$2" ] && echo "ERROR: Server type not provided" && exit 1;
+            SERVER_TYPE="$2"
+            shift 2 ;;
+        *)
+            echo "ERROR: Unknown option provided: $1"
+            exit 1 ;;
+        esac
+    done
+}
+
+parse_args "$@"
+
+# Cleanup
+[ "$do_cleanup" == "true" ] && {
+    systemctl stop sspl-ll ||:
+    yum remove -y cortx-sspl.noarch
+    exit ;
+}
+
+
+# Setup common, 3rd_party and build specific repos
+[ "$setup_repo" == true ] && {
+    curl ${CORTX_MONITOR_BASE_URL}/low-level/files/opt/seagate/sspl/prerequisites/setup_yum_repos.sh -o setup_yum_repos.sh
+    chmod a+x setup_yum_repos.sh
+
+    if [ "$disable_sub_mgr" == "true" ] && [ -n "$TARGET_BUILD" ]; then
+        ./setup_yum_repos.sh -d -t $TARGET_BUILD 2>&1 | tee -a "${LOG_FILE}"
+    elif [ "$disable_sub_mgr" == "true" ]; then
+        ./setup_yum_repos.sh -d 2>&1 | tee -a "${LOG_FILE}"
+    elif [ -n "$TARGET_BUILD" ]; then
+        ./setup_yum_repos.sh -t $TARGET_BUILD 2>&1 | tee -a "${LOG_FILE}"
+    else
+        ./setup_yum_repos.sh 2>&1 | tee -a "${LOG_FILE}"
+    fi
+
+    echo -e "\nDone setup repos" 2>&1 | tee -a "${LOG_FILE}"
+    exit ;
+}
+
+
+# Check for required arguments
+[ -z "$CNTRLR_USER" ] && echo "ERROR: -U <controller user name> is required." && exit 1;
+[ -z "$CNTRLR_PASSWD" ] && echo "ERROR: -P <controller password> is required." && exit 1;
+[ -z "$STORAGE_TYPE" ] && echo "ERROR: --storage_type <type> is required." && exit 1;
+[ -z "$SERVER_TYPE" ] && echo "ERROR: --server_type <type> is required." && exit 1;
+if [ "$PRODUCT_VERSION" == "LDR_R2" ] &&  [ -z "$BMC_IP" ]
+then
+    skip_bmc=true
+    echo "Skipping BMC information for LDR_R2..." 2>&1 | tee -a "${LOG_FILE}" ;
+else
+    [ -z "$BMC_IP" ] && echo "ERROR: --i <BMC IPV4> is required." && exit 1;
+    [ -z "$BMC_USER" ] && echo "ERROR: --u <BMC user> is required." && exit 1;
+    [ -z "$BMC_PASSWD" ] && echo "ERROR: --p <BMC password> is required." && exit 1;
+fi
+
+
+# Install required RPMS if not available
+"$install_3rd_party_packages" && {
+
+    echo "INFO: INSTALLING python3..." 2>&1 | tee -a "${LOG_FILE}"
```

Comment:
python3 will mostly be available on all target systems. yum commands below will just report that its available, so seems fine, anyways its dev deploy script

Reply:
Okay.